### ISSUE TITLE
The Split (allocation) event needs a changeAccount primitive instruction

### ISSUE THREAD

**Issue by mgratacos** [finos/common-domain-model#3129]
## Background ##
Currently Splits (a.k.a allocations) include a partyChange instruction to represent the new counterparty of the allocated trade but it doesn't contain the ability to add/update the account of the "owner" of the allocated trade.

![image (5)](https://github.com/user-attachments/assets/6aabfbf2-5ffd-4cff-a519-89c06999edfe)

![4303b5a7-e16a-408e-bf81-5f7f489e2954](https://github.com/user-attachments/assets/41f25ad6-1a99-4207-93a0-2c649e061b77)

## Proposal ##
We think we would need to add a new accountChangeInstruction that allows adding/updating the account of a party. Please note that currently CDM only allows an optional single account within a party.

More details to follow.

**Comment by chrisisla** [comment]
Hi @mgratacos,

In securities lending we also perform an allocation when a trade is agreed ((and reallocations can be performed after the trade has been confirmed). This is where a single trade is allocating securities from multiple entities - so you could require 50,000 shares for a trade but 10,000 come from fund1, 15,000 from fund2 and the remaining 25,000 from fund3. We also use a Split to perform this.

I can see no problem with adding an accountChangeInstruction to the Split processing, assuming it will take a Party as input. 

You also raise a good point in that the CDM currently only supports one account within a party. Just as an FYI for securities lending we would potentially require 2 (one for cash, one for securities), so this will need to be updated at some point.

Thanks!

**Comment by mgratacos** [comment]
Agreed @chrisisla. The solution for accountChangeInstruction should consider that we may have multiple accounts within a Party at some near future in CDM.

**Comment by ReleaseManagement-BP** [comment]
Closed as this item has been been open and not seen any movement for a considerable amount of time. If you would like the issue re-opened please reach out
